content/learn/intermediate/database.md

```diff
@@ -32,56 +31,48 @@ A database entry has the following structure:
 | .Value     | The value of this entry.                                   |
 | .ValueSize | The size of the value in bytes.                            |
 
-The fields `.CreatedAt`, `.UpdatedAt`, and `.ExpiresAt` all evaluate to a `time.Time` object, so all
-[methods on `time.Time`](/docs/reference/templates/functions#time) are applicable.
+The fields `.CreatedAt`, `.UpdatedAt`, and `.ExpiresAt` all evaluate to a `time.Time` object, so all [methods on `time.Time`](/docs/reference/templates/functions#time) are applicable.
 
 {{< callout context="note" title="Note: About That User ID" icon="outline/info-circle" >}}
 
-The user ID does not have to point to a valid Discord user—it can be any integer. For instance, it is conventional (but
-not required) to store server-global data under the user ID `0`, in which case the `UserID` field will be `0` and the
-`User` field will be invalid. See also [Global vs. User Entries](#global-vs-user-entries).
+The user ID does not have to point to a valid Discord user—it can be any integer. For instance, it is conventional (but not required) to store server-global data under the user ID `0`, in which case the `UserID` field will be `0` and the `User` field will be invalid.
```

Comment:
```suggestion
The user ID does not have to point to a valid Discord user---it can be any integer.
For instance, it is conventional (but not required) to store server-global data under the user ID `0`, in which case the `UserID` field will be `0` and the `User` field will be invalid.
```
wtf is this edge-case?